### PR TITLE
fix: fix `parent_conn` not initialize

### PR DESCRIPTION
**Pull request by Zxilly**
Signed-off-by: Zxilly <zhouxinyu1001@gmail.com>

### CODE REVIEW

**Comment by ffyuanda** [comment]
nb

**Comment by hsluoyz** [comment]
@techoner

File: postgresql_watcher/watcher.py [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
-    port: Optional[int] = 5432,
-    delay: Optional[int] = 2,
-    channel_name: Optional[str] = POSTGRESQL_CHANNEL_NAME,
+        process_conn: Pipe,
+        host: str,
+        user: str,
+        password: str,
+        port: Optional[int] = 5432,
+        delay: Optional[int] = 2,
+        channel_name: Optional[str] = POSTGRESQL_CHANNEL_NAME,
```
Comment (leeqvip) [inline postgresql_watcher/watcher.py]: Incorrect indentation format , it also exists elsewhere

**Review by leeqvip — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@techoner

**Pull request merged**